src/commands/retention-policies/create.js

```diff
@@ -28,6 +28,12 @@ class RetentionPoliciesCreateCommand extends BoxCommand {
 		} else if (flags['non-modifiable']) {
 			options.retention_type = this.client.retentionPolicies.retentionTypes.NON_MODIFIABLE;
 		}
+		if (flags.hasOwnProperty('description')) {
```

Comment:
Can you extend the `RetentionPoliciesCreateCommand.examples` description with new added flags?